src/main/java/internet/shop/services/interfaces/OrderService.java

```diff
@@ -0,0 +1,4 @@
+package internet.shop.services.interfaces;
```

Comment:
```suggestion
package internet.shop.service.interface;
```

same for all other packages that are in plural
let's make them singular

Reply:
interface is not a valid package name so I change it to serviceinterface